htdocs/core/lib/functions.lib.php

```diff
@@ -309,13 +309,15 @@ function getEntity($element, $shared = 1, $currentobject = null)
 		'currentobject' => $currentobject,
 		'out' => $out
 	);
-	$reshook = $hookmanager->executeHooks('hookGetEntity', $parameters, $currentobject, $action); // Note that $action and $object may have been modified by some hooks
+	// avoid from re-initializing results of previous hook (nested hooks)
+	$hookmanager2 = clone $hookmanager;
```

Comment:
The idea of having $hookmanager as a global variable is to have only 1 hook manager instance in memory for all php session.
Here by doing cloning inside getEntity (a very often called method), you may highly increase the memory use.
Can you try a different way to fix:
Inside the executeHooks method, replace the
`$this->resPrint [.]= $actionclassinstance->resprints;`
by
`$localResPrint [.]= $actionclassinstance->resprints;`

and at the end of the method executeHooks:
$this->resPrint = $localResPrint

Does it fix the bug you have ?